packages/patternfly-4/react-core/src/components/Tooltip/Tooltip.d.ts

```diff
@@ -19,9 +19,9 @@ export interface TooltipProps extends Omit<HTMLProps<HTMLDivElement>, 'content'
   /** Tooltip additional class */
   className?: string;
   /** The reference element to which the tooltip is relatively placed to */
-  children: ReactElement<any>;
+  children: React.ReactNode;
```

Comment:
This file should b removed.  We no longer need the d.ts files once we convert the js to tsx files.

Reply:
The Tooltip component hasn't been converted yet, but I had to change this file for type checking.